src/models/ReceiptModel.java

```diff
@@ -0,0 +1,35 @@
+package models;
+
+public class ReceiptModel {
+
+    double sumOfPrices = 0;
```

Comment:
💡 I believe the sumOfPrices is also known as the 'sub-total'  (i.e. before taxes, discounts, coupons, etc are applied)